src/main/java/dk/bec/bookanything/controller/DayOpenController.java

```diff
@@ -0,0 +1,50 @@
+package dk.bec.bookanything.controller;
+
+import dk.bec.bookanything.dto.DayOpenCreateDto;
+import dk.bec.bookanything.dto.DayOpenReadDto;
+import dk.bec.bookanything.mapper.DayOpenMapper;
+import dk.bec.bookanything.model.DayOpenEntity;
+import dk.bec.bookanything.service.DayOpenService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
Suggestion
Change:
@RequestMapping("/api")

To:
@RequestMapping("/api/days-open")